### DISCUSSION TITLE
【灌水】我绳斤来辣！

### DISCUSSION THREAD

**Discussion by SixRadium** [share121/inter-knot#917] [其他]
搞了这么久，终于让我躲过治安局了。
![Uploading G__642_Games_steamapps_common_Genshin Impact Game_ScreenShot_20240619172031.png…]()

**Comment by ZhuYuan-148** [comment]
哦？
![Uploading Screenshot_20240810_180118.jpg…]()

**Comment by Mayf1ower** [comment]
朱鸢警官我是好人，我现在就帮你把他们都抓起来
